src/components/NewCommentForm.tsx

```diff
@@ -57,15 +61,14 @@
       return;
     }
 
-    setSubmitting(true);
+    dispatch(newCommentActions.setNewComment(true));
+    // setSubmitting(true);
 
-    // it is very easy to forget about `await` keyword
     await onSubmit({ name, email, body });
 
-    // and the spinner will disappear immediately
-    setSubmitting(false);
+    dispatch(newCommentActions.setNewComment(false));
+    // setSubmitting(false);
```

Comment:
The `setSubmitting(false)` line is commented out. If you want to stop indicating loading after form submission, consider uncommenting this line.